node-fs-readfile/cat.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+
+for (var i = 2; i < process.argv.length; i++) {
+
+  fs.readFile(`${process.argv[i]}`, 'utf8', (err, data) => {
+    if (err) {
+      console.error(err);
+      return;
+    }
+    console.log(data);
+  });
+}
```

Comment:
This will work to read all files given but it won't guarantee the order they print out in. To complete the challenge the files must print in the order the user supplied them in.